README.md

```diff
@@ -227,6 +227,15 @@ $recommendations = $response->getRecommendation()->getData();
 // }
 ```
 
+You can further modify which items will be reccomended by providing boosting rules. Priority of items matching the
```

Comment:
I'd move this whole added section right before "From $response, you can also access rest of the data:".

Or maybe better merge it with the example following "You can also set more granular options of the recommendation command", something like this:

```php
$recommendation = UserRecommendation::create('user-id', 5, 'test-scenario', 1.0, 3600);
$recommendation->setFilters(['for_recommendation = 1'])
    ->addBoost(Boost::create('valid_to >= NOW()', 2)) // Priority of items matching the Booster criteria will be multiplied by the value of multiplier
    ->addBoost(Boost::create('for_recommendation = 1', 3.5))
    ->setMinimalRelevance(MinimalRelevance::HIGH())
    ->enableHardRotation();

$response = $matej->request()
    ->recommendation($recommendation)
    ->send();
```